packages/elements/src/components/ino-autocomplete/readme.md

```diff
@@ -0,0 +1,83 @@
+# ino-autocomplete
+
+A component that acts similar to the native `datalist` feature of the `<input>` element.
+
+In contrast to other components, this one is stateful, which means that it contains its own state and is therefore less
+flexible to some extent.
+
+This component handles the following tasks:
+
+* Management of the `value` property of the `<ino-input>` element
+* management of showing and hiding the different options filtered on the basis of the input
+* Keyboard navigation on the options
+
+The options are filtered by `.includes(...)`, ignoring upper and lower case.
+
+### Usage
+
+The component can be used as follows:
+
+```html
+
+<ino-autocomplete timeout="<number>">
+  <ino-input slot="input" .../>
+  <ino-list slot="list" ...>
+    <ino-list-item text="Option A"></ino-list-item>
```

Comment:
I see. That's definitely possible. There are two possible solutions that both feel kinda weird:
1. Add a property `value` to `<ino-list-item>` and emit the value on selection
2. Use `<ino-option>` as options

I guess it would be better to use `<ino-option>` as you already suggested. Even though it feelds kinda weird to use `<ino-option>` elements inside an `<ino-list>`